### ISSUE TITLE
"title" attribute cannot be changed dynamically

### ISSUE THREAD

**Issue by AmitMY** [esvit/ng-table#724]
Im trying to create one view for my tables, for I do not want to replecate the same codes over and over.
And I manged to do so, except for the titles of the table (th).

This is my code:
`<td ng-repeat="(k, v) in tableSettings.data" title="'{{v}}'" filter="{ {{k}}: 'text'}" sortable="'{{k}}'">
	{{row[k]}}
</td>`

And this is the error:
Error: [$parse:lexerr] http://errors.angularjs.org/1.4.7/$parse/lexerr?p0=Unterminated%20quote&p1=s%200-4%20%5B'%7B%7Bv%5D&p2='%7B%7Bv ...
**Basically title does not know how to parse my string, for my understanding.

I have tried several versions of ng-table with several versions of angular, but had no luck.
"filter" and "sortable" both works fine, but "title" or "data-title" both crashes the page.

I have tried:
title="'{{v}}'", "{{v}}","v","'v'", and they all failed. (and some also crashed the page)

How can I fix it?

**Comment by c0r3yz** [comment]
I'm having this same issue trying to dynamically set the column header through the title attribute.  Have you gotten anywhere with this yet?

**Comment by c0r3yz** [comment]
This is a duplicated of #468

**Comment by AmitMY** [comment]
Did not get anywhere with it.

**Comment by AmitMY** [comment]
Did anyone get anywhere with it?
